docs/filters.md

```diff
@@ -0,0 +1,53 @@
+# Filters
+
+Both pnpm and Turborepo have a `--filter` argument.
+
+**tl/dr;** Start commands with `pnpm run [cmd]` or `pnpm turbo [cmd]` to use
+Turborepo and its filtering options. See their docs for all options and syntax:
+
+- pnpm docs: [Filtering][1]
+- Turborepo docs: [Filtering Workspaces][2]
+
+## pnpm
+
+To run the test script of a specific package(s), bypassing Turborepo entirely:
+
+```sh
+pnpm --filter <package_selector> <command>
+```
+
+These commands all do the same, the `--filter` argument is provided to `pnpm`:
+
+```sh
```

Comment:
might be worth adding some documentation around the ellipses to select the package and it's dependencies https://pnpm.io/filtering#--filter-package_name-1

Reply:
Good point. I'll add a note, but _really_ trying not to copy existing docs :)